src/mod.rs

```diff
@@ -0,0 +1,54 @@
+// Copyright 2019-2023 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+//! This module contains code that is shared between the library and the build script
+
+use cid::Cid;
+use lazy_static::lazy_static;
+use reqwest::Url;
+
+#[derive(Debug)]
+pub struct ActorBundleInfo {
+    pub manifest: Cid,
+    pub url: Url,
+}
+
+lazy_static! {
```

Comment:
Perhaps we could avoid this macro with `once_cell::sync::Lazy`

Reply:
Fixed